buildSrc/src/main/java/Parser.java

```diff
@@ -168,8 +168,8 @@ private char fixup(int intChar) {
         switch (intChar) {
             case 8217: // back-tick `
                 return '\'';
-            case 8221: // fancy open-quotes ”
-            case 8220: // fancy close-quotes “
+            case 8221: // fancy open-quotes �
+            case 8220: // fancy close-quotes �
```

Comment:
I'm not sure what has happen here, please revert this change.

Reply:
Done. Thanks!